Client/src/Components/Profile/ProfileGraphList.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Grid } from "@material-ui/core"
+
+import { IGraphStateModel } from '../../Models/Graph/IGraphStateModel'
+import { ProfileGraphRow } from '../Profile/ProfileGraphRow'
+import React from 'react'
+import { toGraphDatasetState } from "../../Models/Graph/IGraphDatasetModel"
+
+interface IGDProps {
+    graphDataset: IGraphStateModel[],
+
+}
+
+export const ProfileGraphStateList = (props: IGDProps) => {
+    const { graphDataset } = { ...props }
+
+    const renderGraphListRows = () => {
+        return graphDataset && graphDataset.map(graphDSet => {
+            return (
+                < ProfileGraphRow
+                    graphset={graphDSet}
+                    key={graphDSet.id}
+                    id={graphDSet.id}
```

Comment:
Do you need key and id here? Since they are both mapped to the same value you could remove one

Reply:
> Do you need key and id here? Since they are both mapped to the same value you could remove one

graphSet.id

Reply:
We don't, we were planning on passing the id to a different page but we changed that, I will remove that now, thank you for noticing